src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift

```diff
@@ -5,14 +5,23 @@
 import UIKit
 import OpenCombine
 
+enum pandemicRadarType {
```

Comment:
```suggestion
enum PandemicRadarProvider {
case rki
case bmg
}
```

Reply:
We don't use that type globally, so I guess a private type in an extension of `HomeLinkCardViewModel` would be nice.

Reply:
Good point, I have moved to changed the scope to private and moved to private section.